renovate/flux.json

```diff
@@ -21,13 +21,26 @@
         "hmctsprivate.azurecr.io/camunda/optimize"
       ],
       "enabled": false
+    },
+    {
+      "matchPackageNames": ["hmcts/jenkins-packer"],
```

Comment:
safe to update whenever as it's just updating the agent version for new ones spawning